### ISSUE TITLE
[apex] 

### ISSUE THREAD

**Issue by dpr-nh** [pmd/pmd#4588]
**Affects PMD Version:** 

**Rule:**

Please provide the rule name and a link to the rule documentation:
<https://docs.pmd-code.org/latest/pmd_rules_XXX_XXX.html#XXX>

**Description:**

**Code Sample demonstrating the issue:**

```java

```

**Expected outcome:**

PMD should report a violation at line ..., but doesn't. This is a false-negative.

**Running PMD through:** *[CLI | Ant | Maven | Gradle | Designer | Other]*
